Branch-SDK/src/io/branch/search/DeleteFromSearch.java

```diff
@@ -0,0 +1,35 @@
+package io.branch.search;
+
+import io.branch.indexing.BranchUniversalObject;
+import io.branch.referral.Branch;
+
+/**
+ * Created by sojanpr on 3/2/17.
+ * <p>
+ *     Provides methods to Delete the contents added to Local search
+ *     @see {@link BranchUniversalObject#listOnSamsungSearch()} and {@link SearchBuilder}
+ * </p>
+ */
+public class DeleteFromSearch {
```

Comment:
This class is to hold the deleteFromIndex and deleteAllFromIndex Apis. Don't wanted to add that functionality to Branch.java
usage would be
`DeleteFromSearch.deleteFromSamsungSearch()`